README.md

```diff
@@ -1,303 +1,388 @@
-## Getting Started  
-It is intended and recommended that InSpec and this profile be run from a __"runner"__ host (such as a DevOps orchestration server, an administrative management system, or a developer's workstation/laptop) against the target.
-
-__For the best security of the runner, always install on the runner the _latest version_ of InSpec and supporting Ruby language components.__
-
-Latest versions and installation options are available at the [InSpec](http://inspec.io/) site.
+# Microsoft 365 Foundation CIS Benchmark
+This InSpec Profile was created to facilitate testing and auditing of `CIS Microsoft 365 Benchmark`
+infrastructure and applications when validating compliancy with [Center for Internet Security (CIS) Benchmark](https://www.cisecurity.org/cis-benchmarks)
+requirements.
+ 
+- Profile Version: **3.1.0**
+- Benchmark Date: **2024-04-29**
+- Benchmark Version: **3.0.0**
+ 
+ 
+This profile was developed to reduce the time it takes to perform a security checks based upon the
+CIS Guidance from the Center for Internet Security (CIS).
+ 
+The results of a profile run will provide information needed to support an Authority to Operate (ATO)
+decision for the applicable technology.
+ 
+The CIS Microsoft 365 Foundation CIS CIS Profile uses the [InSpec](https://github.com/inspec/inspec)
+open-source compliance validation language to support automation of the required compliance, security
+and policy testing for Assessment and Authorization (A&A) and Authority to Operate (ATO) decisions
+and Continuous Authority to Operate (cATO) processes.
 
 The M365 CIS Benchmark includes security requirements for an Microsoft 365 environment.
-
-## Getting Started
-
-### Requirements
-
-#### Microsoft 365
+ 
+Table of Contents
+=================
+* [CIS Benchmark  Information](#benchmark-information)
+* [Requirements](#requirements)
+* [Getting Started](#getting-started)
+    * [Intended Usage](#intended-usage)
+    * [Tailoring to Your Environment](#tailoring-to-your-environment)
+    * [Testing the Profile Controls](#testing-the-profile-controls)
+* [Running the Profile](#running-the-profile)
+    * [Directly from Github](#directly-from-github)
+    * [Different Run Options](#different-run-options)
+* [Using Heimdall for Viewing Test Results](#using-heimdall-for-viewing-test-results)
+* [Check Overview]()
+ 
+## Benchmark Information
+The Center for Internet Security, Inc. (CIS®) create and maintain a set of Critical Security Controls (CIS Controls) for applications, computer systems and networks
+connected to the Department of Defense (DoD). These guidelines are the primary security standards
```

Comment:
This is the boilerplate for STIGs but with "CIS Benchmark" stuck in. I'll give you a better paragraph that doesn't reference DOD (CIS isn't a DOD concept like a STIG is).